pkg/config/settings/runtime_setting.go

```diff
@@ -95,3 +96,22 @@ func GetBool(v interface{}) (bool, error) {
 	}
 	return b, nil
 }
+
+// GetInt returns the integer value contained in value.
+// If value is a integer, returns its value
+// If value is a string, it parses the string into an integer.
+// Else, returns an error.
+func GetInt(v interface{}) (int, error) {
+	switch v := v.(type) {
+	case int:
+		return v, nil
+	case string:
+		i, err := strconv.ParseInt(v, 10, 32)
```

Comment:
The idea was to use the smaller size to avoid overflow in `int(i)` later, but `ParseInt` accepts `bitWidth=0` to indicate `int` size, so that's even better.